helm-chart/mezod/values.yaml

```diff
@@ -22,7 +22,10 @@ env:
   MEZOD_PORT_P2P: 26656 # Default. It should be changed in case of a custom setup
 
 # -- Set Secret object containing the keyring information: KEYRING_NAME, KEYRING_PASSWORD, KEYRING_MNEMONIC and ETHEREUM_ENDPOINT
-secret: "NAME_OF_THE_SECRET"
+# -- Set Secret object with node_key.json
```

Comment:
`# -- <comment` is recognized and used by helm-docs.

We should comment separately `credentials` and `node_key`, without any comment for `secret`

Reply:
Addressed!